brianlong/www.validators.app
# 183408173-sol-prices-live

#### What's this PR do?
- create component displaying solana price

#### How should this be manually tested?
- run `rails r daemons/front_stats_update_daemon.rb`
- check component on landing page

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183408173)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
